packages/fds-components/src/components/buttons/GroupButton.md

```diff
@@ -0,0 +1 @@
+`<GroupButton />` should only be used inside `<ButtonGroup />`. This component controls it's own state. See above for example usage.
```

Comment:
`GroupButton` is stateless, so part of this statement may be misleading

Reply:
You think I should just leave off the 2nd sentence? Essentially what I'm trying to communicate, is that ButtonGroup doesn't control any active states for the buttons inside of it... they control their own. That being said, maybe this is obvious looking at the example.

Reply:
I've gone ahead and removed the 2nd sentence.